pkg/materialize/egress_ips.go

```diff
@@ -0,0 +1,15 @@
+package materialize
+
+type EgressIpsBuilder struct {
```

Comment:
We don't actually need a Builder for this since we are just doing the data source query. So I think this just needs the `ReadEgressIpsDatasource` function which is a little silly in this case but probably good to just keep consistency.